src/lib/airtable/tables/__tests__/requests.test.js

```diff
@@ -221,7 +221,6 @@ describe("findDeliveryNeededRequests", () => {
   });
 
   describe("when an error occurs", () => {
-    let result;
```

Comment:
@sandfort added you as a reviewer here since i believe you're helping out with airtable test suites